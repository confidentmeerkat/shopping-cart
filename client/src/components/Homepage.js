import React from 'react';
import FiltersList from './FiltersList';
import Products from './Products';
import '../styles/Homepage.css';

const Homepage = () => (
  <div className="homepage-container">
    <FiltersList />
    <Products />
  </div>
);

export default Homepage;