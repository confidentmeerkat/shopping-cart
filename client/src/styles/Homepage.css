.homepage-container {
  display: flex;
  justify-content: space-between;
  min-height: 100vh;
  margin-top: -72px;
  border: 1px solid #ffffff00;
}