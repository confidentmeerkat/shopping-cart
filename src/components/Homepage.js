import React from 'react';
import FiltersList from './FiltersList';
import Products from './Products';

const Homepage = () => (
  <div className="homepage">
    <FiltersList />
    <Products />
  </div>
);

export default Homepage;