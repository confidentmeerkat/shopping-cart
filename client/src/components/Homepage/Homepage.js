import React from 'react';
import FiltersList from '../FiltersList';
import Products from '../Products';
import '@styles/Homepage.css';

const Homepage = () => (
  <div className="homepage-container">
    <div className="filtersList-desktop">
      <FiltersList />
    </div>
    <Products />
  </div>
);

export default Homepage;