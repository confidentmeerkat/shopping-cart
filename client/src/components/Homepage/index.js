import Homepage from './Homepage';

export default Homepage;
