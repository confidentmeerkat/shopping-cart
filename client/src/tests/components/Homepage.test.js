import React from 'react';
import { shallow } from 'enzyme';
import Homepage from '../../components/Homepage';

describe('<Homepage />', () => {
  it('renders Homepage correctly', () => {
    const wrapper = shallow(<Homepage />);

    expect(wrapper).toMatchSnapshot();
  });
});