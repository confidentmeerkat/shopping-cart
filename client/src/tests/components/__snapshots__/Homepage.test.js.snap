// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Homepage /> renders Homepage correctly 1`] = `
<div
  className="homepage-container"
>
  <div
    className="filtersList-desktop"
  >
    <Connect(FiltersList) />
  </div>
  <Connect(Products) />
</div>
`;
